src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -289,14 +302,23 @@ private void exportDescription(final URI uri) throws FcrepoOperationFailedExcept
             exportLogger.error(String.format("Error exporting description: {}, Cause: {}", uri, ex.getMessage()), ex);
         }
 
-        exportMembers(file);
+        exportMembers(file, uri);
     }
```

Comment:
OK, I've added a blank line — I agree that it's nice to have a blank line between methods, so I'm not sure why there wasn't one.